std_msgs/Bool lamp_1
std_msgs/Bool lamp_2
std_msgs/Bool lamp_3
std_msgs/Bool lamp_4
std_msgs/Bool lamp_5
std_msgs/Bool lamp_6
std_msgs/Bool lamp_7
std_msgs/Bool lamp_8
std_msgs/Bool ok